### DISCUSSION TITLE
allow scaling and maximization of v86 window

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#347] [Feature Requests]
it would be nice if we could shrink the window of the v86 window in the chance that the display size is too large to fit on screen. the way that i would see this working is that when you dragged one of the sides, the other side would move to keep the ratio the same.

**Comment by DustinBrett** [comment]
Thanks for the suggestion! It would indeed be nice to be able to resize v86 but as it stands now it's quite hard. I can adjust the scale similar to what I do with JS-DOS or like you suggested, but the text and image quality degrades a lot I found with v86 as there is no way to tell the resolution in the emulator to change, afaik.
